compact/ecal_barrel_sciglass_tiltless.xml

```diff
@@ -0,0 +1,100 @@
+    <constant name="EcalBarrel_etamax" value="+1.4"/>
+  </define>
+
+  <display>
+    <vis name="det_vis" alpha="1.0" r="1.0" g="0.0" b="0.0" showDaughters="true" visible="false" />
+    <vis name="becal_wedge" alpha="1.1" r="0.82" g="0.84" b="0.86" showDaughters="true" visible="true" />
+    <vis name="carbon_fiber" alpha="0.9" r="0.1" g="0.1" b="0.1" showDaughters="true" visible="true" />
+    <vis name="family1" alpha="1.0" r="0.22" g="0.54" b="0.71" showDaughters="true" visible="true" />
+    <vis name="family2" alpha="1.0" r="0.90" g="0.48" b="0.47" showDaughters="true" visible="true" />
+    <vis name="family3" alpha="1.0" r="0.87" g="0.49" b="0.09" showDaughters="true" visible="true" />
+    <vis name="family4" alpha="1.0" r="0.94" g="0.48" b="0.73" showDaughters="true" visible="true" />
+    <vis name="family5" alpha="1.0" r="0.44" g="0.63" b="0.44" showDaughters="true" visible="true" />
+    <vis name="family6" alpha="1.0" r="0.50" g="0.35" b="0.76" showDaughters="true" visible="true" />
+  </display>
+
+  <detectors>
+    <comment>
+      An EM calorimeter with SciGlass blocks
+    </comment>
+
+    <detector id="4" name="EcalBarrel" type="epic_SciGlassCalorimeterTiltless" readout="EcalBarrelHits" vis="det_vis" calorimeterType="EM_BARREL">
+      <comment>
+        Global detector dimensions from https://eic.jlab.org/Geometry/Detector/Detector-20220624102507.html
+      </comment>
+
+      <dimensions rmin="EcalBarrel_rmin" rmax="EcalBarrel_rmax"
+                  zmax="195 * cm" zmin="-285 * cm"
+                  etamin="EcalBarrel_etamin" etamax="EcalBarrel_etamax"/>
+
+      <wedge-box inner_r="80.0 * cm" thickness="2 * mm" vis="becal_wedge" material="Aluminum5083" />
+
+      <sectors number="24" phi0="0." deltaphi="pi / 12">
+        <comment>
+          Rows within a sector are supposed to have smaller gaps than edge rows of two adjacent sectors
+        </comment>
+
+        <rows number="5" deltaphi="2.92 * degree">
+          <dimensions inner_r="EcalBarrel_rmin + EcalBarrel_inner_margin"  gap="1 * mm" />
+
+          <comment>
+            Families in each row are constructed in two directions `dir_sign` - negative and positive rapidity - starting from the center.
+            The towers have the shape of truncated pyramids that have an opening angle in theta `flare_angle_polar`.
+            The towers are stacked in a row with their edges parallel to the edges of their neighbors, `gap` distance apart.
+            Families of towers are groups of towers having identical shape, i.e. the same tower is repeated `number` amount of times.
+            The front and rear faces are similar isosceles trapezoids only slightly deviating from a rectangular shape by an amount parametrized with `flare_angle_at_face`.
+          </comment>
+
+          <family dir_sign="+1" x1="2 * cm" y1="2 * cm" z_length="45.5 * cm" number="10" flare_angle_polar="1.16 * degree"  flare_angle_at_face="0.3059 * degree" vis="family1" />
```

Comment:
Are `flare_angle_polar` and `flare_angle_at_face` somehow determined by other quantities here?

Reply:
Good question! Both angles should be possible to determine `flare_angle_polar` from numbers of towers per family and azimuthal distribution of towers. In this geometry, we are looking to reproduce an engineering design for which the values of `flare_angle_polar` were hand-optimized for maintaining gaps between towers in azimuthal direction. If calculated, the values may not match the current design, which we are trying to reproduce for the campaign. The current flaring angles are based on a rule that towers are truncated pyramids, which, going forward, is something we might want to abandon anyway. Hence, there is no point to implement that here.

The `flare_angle_at_face` should be easy to constrain, given known `flare_angle_polar` values. That is a TODO.

Reply:
Sounds good.